utils/cronjobs_osgeo_lxd/README.md

```diff
@@ -15,29 +28,31 @@ programmer's manual.
 * generate and deploy the GRASS GIS Web pages at <https://grass.osgeo.org/>:
   * `hugo_clean_and_update_job.sh`
 * GRASS GIS source code weekly snapshots:
-  * grass7-stable: `cron_grass7_relbranch_src_snapshot.sh`
-  * grass8-stable: `cron_grass8_relbranch_src_snapshot.sh`
-  * grass8-devel: `cron_grass8_main_src_snapshot.sh`
+  * grass7-stable (legacy): `cron_grass7_legacy_src_snapshot.sh`
+  * grass8-old-current (current stable): `cron_grass8_old_current_src_snapshot.sh`
+  * grass8-new-current (new stable): `cron_grass8_new_current_src_snapshot.sh`
+  * grass8-preview (development): `cron_grass8_preview_src_snapshot.sh`
```

Comment:
Polished in 15f4a5db51b364e20b2af017e695b5434a2a2794

Reply:
Where does the thing with dashes appear except for this list?

Reply:
It doesn't. It is an "summary" item explanation to not just list the name of the file there.

Better suggestions (please edit directly) are welcome.

Reply:
If I would edit that I would just delete the whole part because the clarity needs to be in the file names not in a possibly outdated list plus-I-don't-get-the-dashes-there, so I pass.